pkg/controller/common/encryptimage.go

```diff
@@ -41,11 +41,11 @@ func UseEncryptedSystemDisk(volume interface{}) (bool, error) {
 	}
 
 	if v, ok := volume.(*extensionsv1alpha1.Volume); ok {
-		return v.Encrypted != nil && *v.Encrypted, nil
+		return (v.Encrypted == nil || v.Encrypted != nil && *v.Encrypted), nil
```

Comment:
Please handle encrypted logic in other logic. There will be side effect when a shoot is reconciled.